components/datatypes/transaction.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/transaction",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Transaction",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:industry": [
```

Comment:
We could also add "retail" here right ? A sales transaction is a common thing in retail Industry.

Reply:
Yes, I've added that.